docs/faq.md

```diff
@@ -0,0 +1,135 @@
+# FAQ
+This is more meant as a "Fully Anticipated Questions" than a "Frequently Asked Questions", as they're anticipated as common pitfalls to using NDE.
```

Comment:
We should have a question that is something like this (rough draft, please improve):

> I am new to Nix. Is this the only way to use doom-emacs with Nix/NixOS?
> No, you can just follow the instructions from doom-emacs website and get a working doom-emacs setup.
> hlissner (the creator of doom-emacs) uses [NixOS](https://github.com/hlissner/dotfiles) and uses a ["normal", imperative setup](https://github.com/hlissner/dotfiles/blob/master/modules/editors/emacs.nix). So should you!